src/components/LabelWithPopover/LabelWithPopover.tsx

```diff
@@ -3,18 +3,14 @@ import type {ReactNode} from 'react';
 import {HelpPopover} from '@gravity-ui/uikit';
 
 interface LabelWithPopoverProps {
-    headerText: string;
-    popoverContent: ReactNode;
+    text: string;
+    content: ReactNode;
```

Comment:
why did you decide to rename this property? As for me, previous name explains what property it is better.

Reply:
I didn't like `headerText` name as it's not a header. But I agree with you regarding `popoverContent` vs just `content`, the first one is better